### ISSUE TITLE
Running into some issues while working with small object detection. 

### ISSUE THREAD

**Issue by andohuman** [experiencor/basic-yolo-keras#205]
I'm trying to count the number of straws from image. I've been using fast rcnn with inception and been getting some good results. But decided to try yolo but not sure how to improve the model from now. 

Obj_thresh = 30%
NMS_thresh = 30%
![20180218_205311_detected](https://user-images.githubusercontent.com/26820345/37842384-f99bfa88-2ee7-11e8-8f3b-8e8b653ad698.jpg)

**Comment by experiencor** [comment]
It looks good. Did you use gen_anchors.py to generate your own anchors? Does faster RCNN works better?

**Comment by andohuman** [comment]
Yeah i used gen_anchors.py to generate the anchors. I used 7 anchor boxes. I first trained the net for 25 epochs heavily penalizing loss_XY then the next 25 epochs havily penalizing the loss_confidence. And this is the best model i've been able to come up with so far. With faster rcnn i am able to get close to 95-100 percent accuracy in counting the straws.

**Comment by experiencor** [comment]
@andohuman I found out this one https://github.com/fizyr/keras-retinanet, which is particularly good for dense object detection like your case. Worth checking out.

**Comment by andohuman** [comment]
@experiencor Yeah, after very little success with Yolo, I tried retinanet and it works like a charm. Thanks for your assistance.

**Comment by experiencor** [comment]
@andohuman Which one is better, RCNN or RetinaNet?

**Comment by andohuman** [comment]
@experiencor I actually haven't had much time to fully work on and understand the retinanet model. As of now the working model im using is faster RCNN and I'm getting pretty good results. How ever the retinanet is showed impressive results the first time I trained. And it is faster than RCNN.

I'm having holidays soon and would be working on retina and see how it goes.
